### ISSUE TITLE
NVDA now echoes keys when in the file search and all other similar dialogs, even when key echo is off

### ISSUE THREAD

**Issue by ahicks92** [microsoft/vscode#146636]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: Insiders commit 8b7086afdbfd42bf6e7b8c0bb586530bf7cf1396
- OS Version: Windows 10

Steps to Reproduce:

1. Start NVDA.
2. Disable key echo, easiest way is NVDA+2 on the laptop keyboard layout, or the keyboard settings.
3. Press ctrl+p or ctrl+shift+p or ctrl+shift+o or anything else that shares that control and start typing.
4. Watch as keys are announced, even when they shouldn't be.
5. Type fast enough and watch as keys are combined into words and announced all at once...

I'm guessing the input element is now inside the live region when it shouldn't be or something like that.  But it is surprisingly awkward to have the screen reader start doing key echo when you have that off and also trying to announce search results at the same time.

**Comment by isidorn** [comment]
@ahicks92 thanks for filling this issue. Was this working before? If yes, when was it working?
I tried to reproduce and I can, even without turning off the NVDA key echo, you can notice that any key typed in the Quick Input(ctr + P) is getting double read. I could not reproduce this with other screen readers so I have filled this issue on the NVDA side https://github.com/nvaccess/nvda/issues/13577

@leonardder might you try to repro this from the NVDA side? 

We are not using using a live region for the Quick Pick.

**Comment by leonardder** [comment]
There's definitely an aria-live polite somewhere. Here is a part of the developer info of the combo box:

```
Developer info for navigator object:
IAccessible accName: 'Search files by name (append : to go to line or @ to go to symbol)'
IAccessible accRole: ROLE_SYSTEM_COMBOBOX
IAccessible accDescription: "Press 'Enter' to confirm your input or 'Escape' to cancel"
IAccessible2 role: ROLE_SYSTEM_COMBOBOX
IAccessible2 attributes: 'display:inline-block;tag:input;xml-roles:combobox;placeholder:Search files by name (append \\: to go to line or @ to go to symbol);autocomplete:list;description-from:aria-describedby;live:polite;relevant:additions text;atomic:false;container-live:polite;container-relevant:additions text;container-atomic:false;container-busy:false;explicit-name:true;haspopup:menu;class:input empty;text-align:left;text-model:a1;text-input-type:text;'
```

As you can see, the live attribute in the IA2 properties is set to polite

**Comment by isidorn** [comment]
@ahicks92 can you please try with VS Code Insiders from Tuesday and this should be fixed. Thanks
